src/cpp/desktop/DesktopGwtCallbackMac.mm

```diff
@@ -460,7 +461,26 @@ bool showOfficeDoc(NSString* path, NSString* appName, NSString* formatString)
    }
 }
 
+void GwtCallback::changeTitleBarColor(int red, int green, int blue) {
+    WId winId = pOwner_->asWidget()->effectiveWinId();
+    if (winId == 0) return;
+    NSView* view = (NSView*)winId;
+    NSWindow* window = [view window];
+    if (pow(red, 2.) + pow(green, 2.) + pow(blue, 2.) < 48387) {
+        window.appearance = [NSAppearance appearanceNamed: NSAppearanceNameVibrantDark];
+    } else {
+        window.appearance = [NSAppearance appearanceNamed: NSAppearanceNameVibrantLight];
+    }
+    if (pow(red, 2.) + pow(green, 2.) + pow(blue, 2.) > 150000) {
```

Comment:
Same for 150000?

Reply:
this is just a random number which works.